### PR TITLE
Update PushNotificationIOS documentation

### PR DESCRIPTION
**Pull request by tanguyantoine**
* add sample for localNotification listener

### CODE REVIEW

File: Examples/UIExplorer/PushNotificationIOSExample.js [hunk 1]
```diff
@@ -43,11 +43,17 @@ var Button = React.createClass({
...
+    // Add listener for push notifications
     PushNotificationIOS.addEventListener('notification', this._onNotification);
+    // Add listener for local notifications
+    //PushNotificationIOS.addEventListener('localNotification', this._onLocalNotification);
   }
 
   componentWillUnmount() {
+    // Remove listener for push notifications
     PushNotificationIOS.removeEventListener('notification', this._onNotification);
+    // Add Remove for local notifications
```
Comment (mkonicek) [inline Examples/UIExplorer/PushNotificationIOSExample.js]: Typo: remove listener

**Comment by mkonicek** [comment]
Instead of `_onLocalNotification`, can you use `_onNotification`? `_onLocalNotification` doesn't exist. Also, how do local notifications get triggered? Can you add some code to the example that actually triggers a local notification? That way you don't need to comment out the code to handle local notifications but can actually show how it works. In general we don't add commented out code to examples.

**Comment by tanguyantoine** [comment]
yep i will do that :)

File: Examples/UIExplorer/PushNotificationIOSExample.js [hunk 2]
```diff
@@ -57,6 +63,11 @@ class NotificationExample extends React.Component {
           onPress={this._sendNotification}
           label="Send fake notification"
         />
+
+        <Button
+          onPress={this._sendLocalNotification}
+          label="Send fake local notification"
```
Comment (mkonicek) [inline Examples/UIExplorer/PushNotificationIOSExample.js]: nit: Please remove 'fake', "Send local notification" reads nicely. The notification is not fake - it is the notification sent by the example.

**Comment by mkonicek** [comment]
Nice!

Please check the flow error on Travis:

    Examples/UIExplorer/PushNotificationIOSExample.js:111
    111:       'Alert message: ' + notification.getMessage(),
                               ^^^^^^^^^^^^ identifier `notification`. Could not resolve name

**Comment by tanguyantoine** [comment]
I update some documentation and nothing is working anymore on circli and travis-ci WTH?

**Comment by mkonicek** [comment]
Tests are currently broken but it's being worked on.

@facebook-github-bot shipit

**Comment by mkonicek** [comment]
(`@facebook-github-bot shipit` will still run internal tests)

**Pull request closed without merging**